source/plugin/ray-tracing.rst

```diff
@@ -62,7 +63,7 @@ Finally, use :javadoc:`BlockRay.BlockRayBuilder#build()` to finish building the
 
     Player player;
     BlockRay<World> blockRay = BlockRay.from(player)
-        .stopFilter(BlockRay.continueAfterFilter(BlockRay.onlyAirFilter(), 1)).build();
+        .skipFilter(BlockRay.onlyAirFilter()).stopFilter(BlockRay.onlyAirFilter()).build();
```

Comment:
So, this line skips air, then stops if it _doesn't_ find air, correct? In this case, the lines and code block below can just be removed - a `stopFilter` that is an `allFilter` will never stop... right?

Not one for the docs, but you know, I'm really tempted to add names that make more sense to the API for 7.2. Thinking about this is making me go insane.